src/components/SettingsDialog/Participants/Participants.scss

```diff
@@ -1,90 +1,232 @@
 @import "src/constants/style";
 @import "../SettingsDialog";
 
-.participants__container {
-  display: flex;
-  flex-direction: column;
-  height: auto;
-  margin-bottom: 62px;
-  overflow: auto;
-  flex-grow: 1;
+.participants__search-input-wrapper {
+  position: relative;
+  height: 40px;
+  width: 100%;
 }
-
-.participants__user-list-wrapper {
-  overflow: auto;
+.participants__search-input {
+  height: 100%;
   width: 100%;
-  // overflow-x: hidden; // firefox bug?
+  padding: 0 calc($padding--default + $icon--medium + $padding--small) 0 $padding--default;
+  font-size: 14px;
+  font-weight: bold;
+  letter-spacing: $letter-spacing--medium;
+  color: $color-black;
+  background-color: $color-white;
+  border: none;
+  border-radius: $rounded--full;
 
-  @include scrollbar();
+  &:focus {
+    outline: 2px solid var(--accent-color);
+  }
+  &:hover {
+    background-color: var(--accent-color--desaturated-light);
+  }
+}
+.participants__search-icon {
+  position: absolute;
+  height: 100%;
+  width: $icon--medium;
+  color: $color-black;
+  right: $padding--default;
 }
 
-.participants__user-list {
+.participants__filter-buttons {
+  margin-top: $margin--small;
   display: flex;
-  flex-direction: column;
-  width: 100%;
-  border-radius: 8px;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.participants__permisson-filter-button {
+  height: 42px;
   background-color: $color-white;
+  color: $color-black;
+  border: none;
+  border-radius: $rounded--full;
+  padding: 0 10px;
+  font-weight: bold;
+  font-size: 14px;
```

Comment:
The `style.scss` has `$text-size--small` and `$text-size--medium` for 12px and 16px if you want to use them.

Reply:
I've used the font sizes and line heights specified by the design